### ISSUE TITLE
Chapter 3: [Classification] Unable to download MNIST data

### ISSUE THREAD

**Issue by cabudies** [ageron/handson-ml#290]
As per the code,
`mnist = fetch_mldata('MNIST original')`
it should download the code. However, it's not happening. Also, I have downloaded the training set separately but then how to go with the test set? Unable to view images that have been downloaded in the training set.

**Comment by piraka9011** [comment]
Check this issue out #143

**Comment by ageron** [comment]
Hi @cabudies , as @piraka9011 mentioned, you will find the answer in #143.

**Comment by ageron** [comment]
If anyone still has issues downloading MNIST, please check out the fix in #301.
